### ISSUE TITLE
Identical looking items should behave identical

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#660]
On a video page the “Embed” and “Manage” buttons look identical and you would expect them to behave identical.:

![Screenshot from 2023-01-12 22-58-55](https://user-images.githubusercontent.com/1008395/212192600-500c141e-07be-4f4c-b558-abe1a81c11a9.png)

That is, unless you use a screen reader which will tell you something along the lines of:

- Manage visited link
- Embed push button

If we want them to look alike and they should feel alike to users, we should use the same components and not style one component to look like a completely different one.

**Comment by owi92** [comment]
This can be closed. The colour of the `manage` button was changed to a "link colour" to indicate that it is serving a different purpose than the other buttons. While not everyone is convinced that this solution is sufficient, we talked about this several times and came to the conclusion that we'll stick with it for now.
